app/components/jobseekers/search_results/heading_component.rb

```diff
@@ -4,25 +4,26 @@ def initialize(vacancies_search:)
     @keyword = @vacancies_search.keyword
     @location = @vacancies_search.location_search.location
     @polygon_boundaries = @vacancies_search.location_search.polygon_boundaries
+    @radius = @vacancies_search.search_criteria[:buffer_radius] || @vacancies_search.search_criteria[:radius]
```

Comment:
I thought you were getting rid of buffer_radius?

Reply:
Well spotted!

Reply:
I think this got through in the confusion of splitting PRs and rebasing etc etc